Bootstraps/Symfony.php

```diff
@@ -49,16 +49,24 @@ public function getApplication()
         $appAutoLoader = './app/autoload.php';
         if (file_exists($appAutoLoader)) {
             require $appAutoLoader;
-        } else {
+        } elseif (file_exists('./vendor/autoload.php')) {
             require './vendor/autoload.php';
         }
+        if(class_exists('\AppKernel')) {
+            $app = new \AppKernel($this->appenv, $this->debug);
+        } else {
+            if(file_exists('./app/bootstrap.php.cache') && file_exists('./app/AppKernel.php')) {
```

Comment:
Mh, interesting. In my Symfony 2.8 everything works correctly. I used http://symfony.com/doc/current/book/installation.html. How have you installed Symfony 2.8?